<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReactStudy</class>
 <widget class="QWidget" name="ReactStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>reactStudy 1.0</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QGroupBox" name="stateGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="title">
    <string>运行状态</string>
   </property>
   <widget class="QLabel" name="stateText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>note:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="stateColor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="aggGroupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="title">
    <string>更新/构建</string>
   </property>
   <widget class="QPushButton" name="updateBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>构建</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="switchGroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="title">
    <string>react 启停</string>
   </property>
   <widget class="QPushButton" name="startBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="text">
     <string>重启</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="infoGroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="title">
    <string>提示信息</string>
   </property>
   <widget class="QTextEdit" name="msgView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>配置路径</string>
   </property>
   <widget class="QPushButton" name="pathConfigBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="text">
     <string>路径配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="text">
     <string>端口配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>startBtn</tabstop>
  <tabstop>stopBtn</tabstop>
  <tabstop>restartBtn</tabstop>
  <tabstop>updateBtn</tabstop>
  <tabstop>buildBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickStartBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickStopBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restartBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickRestartBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickBuildBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickUpdateBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathConfigBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReactStudy</receiver>
   <slot>clickPathConfigBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickStopBtn()</slot>
  <slot>clickStartBtn()</slot>
  <slot>clickRestartBtn()</slot>
  <slot>clickPathConfigBtn()</slot>
  <slot>clickBuildBtn()</slot>
  <slot>clickUpdateBtn()</slot>
 </slots>
</ui>
